models/QtiRunnerInitDataBuilder.php

```diff
@@ -123,6 +123,34 @@ public function build($deliveryExecutionId, $withScores = false)
         return $init;
     }
 
+    public function getQtiTestItems(string $deliveryExecutionId): array
```

Comment:
Can we add unit tests here?

Reply:
I've moved this logic here https://github.com/oat-sa/extension-tao-outcome/pull/242/files#diff-21357e0b8799d4075614d38401cd14f2e765bbab11421ceac475cedb3b43e7d7 and created test here https://github.com/oat-sa/extension-tao-outcome/pull/242/files#diff-6ffe142eaddf3b73c81cffe82809d046f3a37228737e99f6a60a492c7ca97fbc